test/lib/util.js

```diff
@@ -432,7 +477,25 @@ async function setupAliceAndBob({ aliceOptions, bobOptions }) {
   const aliceRemote = bobRoom.participants.get(aliceLocal.sid);
   const bobRemote = aliceRoom.participants.get(bobLocal.sid);
 
-  return { aliceRoom, bobRoom, aliceLocal, bobLocal, aliceRemote, bobRemote, roomSid, roomName };
+  const nTracks = Math.min(...[aliceLocal, bobLocal].map(participant => participant.tracks.size));
+  await waitFor([aliceRemote, bobRemote].map(participant => tracksSubscribed(participant, nTracks)), 'tracks to be subscribed');
```

Comment:
why wait for minTracks to be subscribed? we should either wait for correct number of tracks for each participant or not wait at all.